app/i18n/es.json

```diff
@@ -202,7 +202,10 @@
         "ref_volume": "Volumen de referidos",
         "ref_bonus": "Bonificación de referidos",
         "ref_commission": "Comisión de referidos",
-        "change": "Cambiar la dirección"
+        "change": "Cambiar la dirección",
+        "vip_title": "Welcome to the Black Squad",
+        "vip_description": "The Black Squad is an elite group for major investors on DFX and gives you a direct voice channel to DFX leadership.",
+        "vip_link": "The Black Squad gathers in a Telegram group - just follow the link:"
```

Comment:
No translation (also FR, IT, PT)?

Reply:
Yes. Just DE and EN (for all others) for the first step.